### PR TITLE
make condition of DumpGraphToFile in jit passes be same

### PR DESCRIPTION
**Pull request by LogX2**
When i debugged the xla module, i found the 3 passes in jit used different conditions to dump their graphs. So here i modified the condition of DumpGraphToFile in MarkForCompilationPass in order to 
 align with EncapsulateSubgraphPass and BuildXlaOpsPass.

### CODE REVIEW

**Review by sanjoy — CHANGES_REQUESTED** [review]
Is there a specific problem with the current scheme?  I don't want to break users' workflow without good reason.

I'd be fine if you want to change the condition to `debug_options_.dump_graphs || VLOG_IS_ON(1)`, although I think it should be `VLOG_IS_ON(3)` at least to avoid spamming the temp directory with files.

Also, running a binary as `TF_DUMP_GRAPH_PREFIX=/tmp/path/to/graph blaze-bin/.../binary --vmodule=optimization_registry=1` will dump TensorFlow graphs after every `GraphOptimizationPass` (including `MarkForCompilationPass`) to `/tmp/path/to/graph`.

File: tensorflow/compiler/jit/mark_for_compilation_pass.cc [hunk 1]
```diff
@@ -643,7 +643,7 @@ Status MarkForCompilationPassImpl::CreateClusters() {
   // Names for each cluster.
   std::unordered_map<int, string> cluster_names;
 
-  if (debug_options_.dump_graphs) {
+  if (debug_options_.dump_graphs || VLOG_IS_ON(1)) {
```
Comment (sanjoy) [inline tensorflow/compiler/jit/mark_for_compilation_pass.cc]: What do you think about making this `VLOG_IS_ON(3)`?

Comment (LogX2) [reply tensorflow/compiler/jit/mark_for_compilation_pass.cc]: I am agree with your idea to reduce the debug level  of dump file , but i am not sure which specific level to reduce to . Is there any instrument docs for this?

**Review by sanjoy — DISMISSED** [review]
lgtm

**Comment by sanjoy** [comment]
@rthadur I thought I already approved this CL internally.  Did that not go through?

**Comment by rthadur** [comment]
@sanjoy there were new changes pushed by @LogX2  so we need your approval again , thank you

**Comment by LogX2** [comment]
sorry to disturb you, it is caused by another code change in a wrong branch and have been  corrected after i realized that.

**Comment by rthadur** [comment]
@LogX2 there are still conflicts , can you please resolve them.

File: tensorflow/compiler/jit/increase_dynamism_for_auto_jit_pass.cc [hunk 2]
```diff
@@ -374,14 +374,14 @@ Status FindAndRewriteSlices(Graph* g, bool* changed) {
 Status IncreaseDynamismForAutoJitPass::Run(
     const GraphOptimizationPassOptions& options) {
   MarkForCompilationPassFlags* flags = GetMarkForCompilationPassFlags();
-  if (flags->tf_xla_clustering_debug) {
+  if (flags->tf_xla_clustering_debug||VLOG_IS_ON(3)) {
```
Comment (sanjoy) [inline tensorflow/compiler/jit/increase_dynamism_for_auto_jit_pass.cc]: Please clang-format.

**Review by sanjoy — DISMISSED** [review]
lgtm

**Pull request closed without merging**